docsrc/allblocks.nim

```diff
@@ -104,6 +127,21 @@ Here is the source code and the centered title:
 nimibCode:
   nbRawHtml: """<h4 style="text-align: center">Centered title</h4>"""
 
+nbCodeBlock: "nbShow"
+nbText: """
+Some Nim objects have been designed with an HTML renderer. If the object has an associated `toHtml()` procedure, it can be rendered with the nbShow plot.
```

Comment:
```suggestion
Some Nim objects have been designed with an HTML renderer. Nimib allows to pretty-print objects by rendering them to HTML. If the object has an associated `toHtml()` procedure, it can be rendered with `nbShow`.
```
While the first sentence is technically true, it is not the point of the `nbShow` block. This is a suggestion on how to rephrase it, but feel free to modify it :D.

Reply:
Dropping entirely the first sentence seems better.
```suggestion
Nimib allows to pretty-print objects by rendering them to HTML. If the object has an associated `toHtml()` procedure, it can be rendered with `nbShow`.
```

Reply:
I think the first sentence is nice, it gives a brief description of the block instead of jumping right in.

Reply:
Ah yes sorry, I forgot to remove your original first sentence 😅 So yes, you are correct, it can be removed 👍